tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -1,107 +1,34 @@
-      logProjectEvents: false
-      workingFolder: '$(Build.BinariesDirectory)\$(BuildConfig)'
-      createLogFile: true
-
-  - template: templates/set-test-data-variables-step.yml
-
-
-  - task: BatchScript@1
-    displayName: 'Setup VS2019 env vars'
-    inputs:
-      filename: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat'
-      arguments: $(buildArch)
-      modifyEnvironment: true
-
-  - script: |
-     set PATH=$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig);%PATH%
-
-  - task: VSTest@2
-    displayName: 'VsTest - testAssemblies'
-    inputs:
-      testAssemblyVer2: 'onnxruntime_test_all.exe'
-      searchFolder: '$(Build.BinariesDirectory)\Debug\Debug'
-      runSettingsFile: 'cmake\\codeconv.runsettings'
-      pathtoCustomTestAdapters: '$(Build.BinariesDirectory)\Debug\googletestadapter.0.17.1\build\_common'
-      codeCoverageEnabled: true
-      platform: x64
-
-  - task: PublishTestResults@2
-    inputs:
-      searchFolder: '$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)\TestResults'
-      testResultsFormat: 'VSTest'
-      testResultsFiles: '**'
-    condition: succeededOrFailed()
-
-  - template: templates/component-governance-component-detection-steps.yml
-    parameters :
-      condition : 'succeeded'
-
-  - template: templates/clean-agent-build-directory-step.yml
-
+    - template: templates/set-version-number-variables-step.yml
+
+    - task: CmdLine@2
+      inputs:
+        script: |
+          set -e
+          sudo apt-get install -y python3-pip
+          sudo python3 -m pip install mysql-connector-python
+          ln -s /data/models .
+          LLVM_PROFILE_FILE="%p.profraw" CFLAGS="-g -fprofile-instr-generate -fcoverage-mapping" CXXFLAGS="-g -fprofile-instr-generate -fcoverage-mapping" CC=clang CXX=clang++  python3 $(Build.SourcesDirectory)/tools/ci_build/build.py --build_dir=$(Build.BinariesDirectory) --config Debug --parallel --skip_submodule_sync --build_shared_lib --enable_onnx_tests
+          cd Debug
+          LLVM_PROFILE_FILE="testrunner.profraw" ./onnx_test_runner -j 1 -c 2 /data/models
+          llvm-profdata merge -sparse  -o ort.profdata *.profraw
+          llvm-cov export -summary-only -instr-profile=ort.profdata onnx_test_runner -object onnxruntime_test_all  $(Build.SourcesDirectory)/include/onnxruntime $(Build.SourcesDirectory)/onnxruntime/core $(Build.SourcesDirectory)/onnxruntime/contrib_ops > $(Build.BinariesDirectory)/report.json
+        workingDirectory: $(Build.BinariesDirectory)
+
+
+    - task: CmdLine@2
+      displayName: 'Post code coverage data to the Dashboard'
+      inputs:
+        script: |
+          python3 $(Build.SourcesDirectory)/tools/ci_build/github/windows/post_code_coverage_to_dashboard.py  --commit_hash=$(OnnxRuntimeGitCommitHash) --report_file="$(Build.BinariesDirectory)/report.json" --report_url="https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=$(Build.BuildId)"
```

Comment:
>"https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=$(Build.BuildId)" [](start = 220, length = 87)

Is this consumed by a dahsboard? Can we put more comments into files?

Reply:
Yes. I'll will.